third_party/move/move-compiler-v2/tests/testsuite.rs

```diff
@@ -36,8 +36,8 @@ pub const EXP_EXT: &str = "exp";
 /// Configuration for a set of tests.
 #[derive(Default)]
 struct TestConfig {
-    /// Whether only type check should be run.
-    type_check_only: bool,
+    /// Whether stackless bytecode passes should be skipped.
+    skip_bytecode_passes: bool,
```

Comment:
I am not sure if this is accurate: this flag also skips file format generation, in addition to the stackless bytecode pipeline. Maybe `stop_before_sb_pipeline` or `stop_before_sb_gen` or similar?